apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java

```diff
@@ -348,19 +394,31 @@ private StaticRifResource(
     }
   }
 
-  /** @return the {@link URL} to the resource's contents */
+  /**
+   * Get Resource Url as {@link URL}.
```

Comment:
Not what I meant. Should be:
```suggestion
   * Gets the {@link #resourceUrl}.
```
link to the documentation of the field by referencing the field in the doc. (The [style guide](https://github.com/CMSgov/beneficiary-fhir-data/blob/8bb739ca6ff4ac8e2e9c6581d2b52f941b981d80/docs/style/style-guide.md) is updated with this info now.)